webclient/index.html

```diff
@@ -125,6 +139,8 @@
                 alert('Whoops. The cached files on your machine are out of sync with each other. That\'s our bad. Please hard-refresh the page?');
             }
         </script>
+<flip-card id="mainFlipCard">
```

Comment:
I haven't seen it yet, but only flipping the right side seems like a better option